channel/ledger/channel.go

```diff
@@ -0,0 +1,244 @@
+package ledger
+
+import (
+	"fmt"
+	"math/big"
+	"sort"
+
+	"github.com/statechannels/go-nitro/channel/state"
+	"github.com/statechannels/go-nitro/channel/state/outcome"
+	"github.com/statechannels/go-nitro/types"
+)
+
+const proposerIndex = uint(0)
+
+// ConsensusChannel is used to manage states in a running ledger channel
+type ConsensusChannel struct {
+	// constants
+	Id             types.Destination
+	MyIndex        uint
+	OnChainFunding types.Funds
+	state.FixedPart
+
+	// variables
+	current     SignedVars       // The "consensus state", signed by both parties
+	proposalQue []SignedProposal // A queue of proposed changes, starting from the consensus state
```

Comment:
I think these fields would need to get stored in the store? So we'd need to add serde for them.

Reply:
Yes